ui/src/app/shared/header/header.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="content">
+  <md-form-field class="query-form">
+    <md-chip-list #chipList>
+      <md-chip id="chip" *ngFor="let chipKey of getChipKeys()"
+               [mdMenuTriggerFor]="menu"
+               (remove)="removeChip(chipKey)"
+               (click)="setCurrentChip(chipKey)">
+        {{ getDisplayValue(chipKey) }}
+        <img mdChipRemove
+             src="https://www.gstatic.com/images/icons/material/system/1x/cancel_grey600_18dp.png">
+      </md-chip>
+      <md-menu #menu="mdMenu" [overlapTrigger]="false" class="wide-menu">
+        <md-input-container class="query-input">
+          <input mdInput
+                 placeholder="Equals"
+                 type="search"
+                 [(ngModel)]="currentChipValue"
+                 (keyup.enter)="assignChipValue($event)"
+                 (click)="$event.stopPropagation()"/>
+        </md-input-container>
+      </md-menu>
+      <input mdInput
+             placeholder="Filter jobs..."
+             [(ngModel)]="inputValue"
+             value="{{inputValue}}"
+             [mdAutocomplete]="auto"
+             [mdChipInputFor]="chipList"
+             [mdChipInputSeparatorKeyCodes]="separatorKeysCodes"
+             [mdChipInputAddOnBlur]="false"
+             (mdChipInputTokenEnd)="addChip($event.value)"
+             [formControl]="control"/>
+      <md-autocomplete #auto="mdAutocomplete">
+        <md-option
+          *ngFor="let option of filteredOptions | async"
+          [value]="option"
+          (click)="addChip(option)">
```

Comment:
Is there a way to make it so that when you select an auto-completed key and the chip is created it goes directly to the input form to define what the value equals? It's a little annoying now to select a chip key from the list then have to click on it again to input the desired value.

Reply:
SGTM